x/evm/keeper/msg_server.go

```diff
@@ -1459,7 +1462,7 @@ func (s msgServer) SignCommands(c context.Context, req *types.SignCommandsReques
 
 	commandList := types.CommandIDsToStrings(commandBatch.GetCommandIDs())
 	for _, commandID := range commandList {
-		s.Logger(ctx).Debug("signing command batch", "commandBatchID", batchedCommandsIDHex, "commandID", commandID)
+		s.Logger(ctx).Debug("signing command %s in batch %s", commandID, batchedCommandsIDHex)
```

Comment:
why take out from attribute?

Reply:
For almost everything else we put the info in the log message so it's easy to search it. e.g. put a command batch hex in search query, and all corresponding logs show up. Makes it much easier to debug instead of searching by message in some places and by attributes in others.

Reply:
can search use `facet` though?

Reply:
I feel like using attributes should be the way to go, perhaps gradually, but feel free to ignore if you disagree.

Reply:
Agree we should be using facets but given that we pay per new line character is there a downside to including information in the message content? It makes cross-referencing IDs between services much easier.

But data like this should always at minimum be in the attributes, not exclusively in the message content.

Using facets effectively requires us to have 
1. comprehensive set of query templates including the facet names (no one can remember them all) 
2. consistent attribute keys (facet names) 

We don't have either at this stage - we need to do a log audit imo

Reply:
Yep, logs should be reworked properly at some point, so we have consistency for now.

Reply:
https://github.com/axelarnetwork/axelar-core/issues/1227